lib/browser/rpc-server.js

```diff
@@ -274,6 +275,15 @@ const handleRemoteCommand = function (channel, handler) {
   })
 }
 
+const emitCustomEvent = function (contents, eventName, ...args) {
+  const event = eventBinding.createWithSender(contents)
+
+  electron.app.emit(eventName, event, contents, ...args)
```

Comment:
What for?

Reply:
It's applying here automatically the event forwarding step that was removed from `web-contents.js`. Will we always want to forward _any_ custom events to `app`? Not sure if the purpose of keeping the `forwardedEvents` array was to keep a list of events to forward, while others wouldn't be forwarded.

Reply:
@alexeykuzmin 
- check the other PR that depends on this, I need to be able to check if there are no listeners, which is much easier to do with this change
- it's cleaner like this and also there is internal listener used, which could be screwed up by the app code theoretically

Reply:
if we need to change the behavior in the future, we will just change the code to do whatever might be needed at the time.